revive.toml

```
@@ -19,6 +19,9 @@ warningCode = 1
 [rule.error-strings]
 [rule.errorf]
 [rule.exported]
+[rule.filename-format]
+    # Override the default pattern to forbid .go files with uppercase letters, underscores, and dashes.
```

```suggestion
    # Override the default pattern to forbid .go files with uppercase letters and dashes.
```

Underscores are in the pattern, and you are renaming files to have underscores instead of dashes

---

My bad. Fixed. Thank you.